### DISCUSSION TITLE
Hi ..can we hide browse library link ?? that is comes under library

### DISCUSSION THREAD

**Discussion by sohailamjad256** [excalidraw/excalidraw#4487] [Q&A] [answered]
hey..i just want to hide browse library option which is comes under library..Thanks

**Comment by ad1992** [comment, marked as answer]
Its not supported via any API yet, you will have to hide it via css

**Reply by ad1992** [reply]
@sohailamjad256 not sure if your 👇🏻  comment was deleted as I am not able to find it
> okay but i don't see the codes..can you tell me the class names or any id??

Since it doesn't have any class name/id you can use target selector to hide it
```css
a[target="_excalidraw_libraries"] {
  display:none;
}
```

https://user-images.githubusercontent.com/11256141/147535130-1ac9e06b-836b-4c06-9847-5c54e2f9697c.mov

**Reply by sohailamjad256** [reply]
Hi..Thanks for the help...actually this one is also working for me...
layer-ui__library-header a {
  display: none;
}

**Reply by ad1992** [reply]
> Hi..Thanks for the help...actually this one is also working for me... layer-ui__library-header a { display: none; }

I would suggest not to use this one as its targeting all anchor tags inside library header which will break if tomorrow one more link is added which you might not want, hence targeting specific selector is safer.

**Reply by sohailamjad256** [reply]
okay..then i will use target ..
Thanks @ @ad1992

**Comment by sohailamjad256** [comment]
Hi @ad1992  
can we change excalidraw toolbar color? inplace of purple can we give different color

**Reply by ad1992** [reply]
yep as mentioned by @dwelle [here](https://github.com/excalidraw/excalidraw/discussions/4615#discussioncomment-1982938)
